src/dns_operations/mod.rs

```diff
@@ -297,13 +297,13 @@ mod test {
         let dns_name = eval_result!(::safe_client::utility::generate_random_string(10));
         let messaging_keypair = ::sodiumoxide::crypto::box_::gen_keypair();
 
-        let mut services = vec![("www".to_string(),     (::routing::NameType::new([123; 64]), 15000)),
-                                ("blog".to_string(),    (::routing::NameType::new([124; 64]), 15000)),
-                                ("bad-ass".to_string(), (::routing::NameType::new([124; 64]), 15000))];
+        let mut services = vec![("www".to_string(),     ::safe_nfs::metadata::directory_key::DirectoryKey::new(::routing::NameType::new([123; 64]), 15000, false, ::safe_nfs::AccessLevel::Private)),
```

Comment:
<1> makes sense to break it down in multiple lines:
```
::safe_nfs::metadata::directory_key::DirectoryKey::new(::routing::NameType::new([123; 64]),
                                                                                        5000,
                                                                                        false,
                                                                                        ::safe_nfs::AccessLevel::Public)),
::safe_nfs::.....
```
<2> Also make the access level as Public (as DNS usually deals with Public data - the browser will not be able to decode servic home-pages otherwise).